components/breadcrumbs/README.md

```diff
@@ -28,124 +28,120 @@ Breadcrumbs are a way-finding tool that helps users understand where they are w
+  <d2l-breadcrumb text="Item 2" href="#"></d2l-breadcrumb>
+  <d2l-breadcrumb text="Item 3" href="#"></d2l-breadcrumb>
 </d2l-breadcrumbs>
 ```
 
-This works well for mobile or other touch devices but not as well for mouse or keyboard users, so we have two other options for managing width.
+<!-- docs: start hidden content -->
+### Properties
 
-### Limited Width
+| Property | Type | Description |
+|--|--|--|
+| `compact` | Boolean | Indicates whether the component should render in compact mode |
+<!-- docs: end hidden content -->
 
-Set a `max-width` to constrain breadcrumbs to a particular width:
+## Breadcrumb (child) [d2l-breadcrumb]
 
-<!-- docs: demo code display:block -->
+<!-- docs: demo code properties name:d2l-breadcrumb sandboxTitle:'Breadcrumb' display:block -->
 ```html
 <script type="module">
   import '@brightspace-ui/core/components/breadcrumbs/breadcrumbs.js';
 </script>
-<d2l-breadcrumbs style="max-width: 250px">
-  <d2l-breadcrumb text="Trucate Basic Item 1" href="#"></d2l-breadcrumb>
-  <d2l-breadcrumb text="Truncate Basic Item 2" href="#"></d2l-breadcrumb>
+<d2l-breadcrumbs>
+  <d2l-breadcrumb text="Item 1" href="#"></d2l-breadcrumb>
 </d2l-breadcrumbs>
 ```
 
-### Compact Mode
+<!-- docs: start hidden content -->
+### Properties
 
-Alternately, add the `compact` attribute to only display the last breadcrumb. The `d2l-breadcrumb-current-page` will be hidden:
+| Property | Type | Description |
+|--|--|--|
+| `text` | String, required | Text of the breadcrumb item |
+| `aria-label` | String | AriaLabel of breadcrumb item |
+| `href` | String | Href of the breadcrumb item |
+| `target` | String | Target of the breadcrumb item |
+<!-- docs: end hidden content -->
 
-<!-- docs: demo code display:block -->
+## Current Page [d2l-breadcrumb-current-page]
+
+Only include the current page in the breadcrumb if your page or view does not have a visible heading. You will notice that some older pages or tools in Brightspace still display the current page as the last breadcrumb despite having a visible page heading, but this is now a legacy pattern.
+
+<!-- docs: demo code properties name:d2l-breadcrumb-current-page sandboxTitle:'Current Page Breadcrumb' display:block -->
 ```html
 <script type="module">
   import '@brightspace-ui/core/components/breadcrumbs/breadcrumb-current-page.js';
   import '@brightspace-ui/core/components/breadcrumbs/breadcrumbs.js';
 </script>
-<d2l-breadcrumbs compact>
+<d2l-breadcrumbs>
   <d2l-breadcrumb text="Item 1" href="#"></d2l-breadcrumb>
   <d2l-breadcrumb text="Item 2" href="#"></d2l-breadcrumb>
   <d2l-breadcrumb-current-page text="Current Page"></d2l-breadcrumb-current-page>
 </d2l-breadcrumbs>
 ```
 
-## Breadcrumbs [d2l-breadcrumbs]
+## Responsive Behavior
 
-<!-- docs: demo code properties name:d2l-breadcrumbs sandboxTitle:'Breadcrumbs' display:block -->
+Breadcrumbs that overflow their container will appear to fade at the edge, as in this example:
```

Comment:
While things were shifted around, the line `This works well for mobile or other touch devices but not as well for mouse or keyboard users, so we have two other options for managing width.` Seems to have been removed, so I just wanted to verify if that was intentional or not.

Reply:
Yeah, it was on purpose. That statement was odd and not 100% correct, so I removed it.